src/Umbraco.Infrastructure/Services/ContentListViewServiceBase.cs

```diff
@@ -90,17 +90,18 @@ protected ContentListViewServiceBase(TContentTypeService contentTypeService, IDa
             .Select(p => p.Alias)
             .WhereNotNull();
 
-        // Service layer expects "owner" instead of "creator", so make sure to pass in the correct field
-        if (orderBy.InvariantEquals("creator"))
-        {
-            orderBy = "owner";
-        }
 
         if (listViewPropertyAliases.Contains(orderBy) == false && orderBy.InvariantEquals("name") == false)
         {
             return Attempt.FailWithStatus<Ordering?, ContentCollectionOperationStatus>(ContentCollectionOperationStatus.OrderByNotPartOfCollectionConfiguration, null);
         }
 
+        // Service layer expects "owner" instead of "creator", so make sure to pass in the correct field
+        if (orderBy.InvariantEquals("creator"))
+        {
+            orderBy = "owner";
+        }
+
```

Comment:
Why isn't just the right orderBy passed in? 🧂

Reply:
Because the service layer expects "owner" instead of "creator", so we need to make sure to pass in the correct field 🙂

Reply:
I mean why is there a difference between these two, why isn't just "owner" passed in this method instead of "creator".

Reply:
Because we are migrating from "owner" on the front end in favour of "creator".